unit_tests/README.md

```diff
@@ -0,0 +1,8 @@
+# Unit Tests
+
+The scope of the unit tests is to test separate, distinct features of MAUS: units.
+
+The data on which they rely on are mostly made up "artificial" data or real data stripped down to the essential part
```

Comment:
```suggestion
The data they rely on are mostly made up "artificial" data or real data stripped down to the essential part
```

Reply:
Reading this sentence I entered the world of the data is vs data are discussion:
I like dthis article :-)
https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular